src/DropdownFilter.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.DropdownFilter.DropdownFilter" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Dropdown filter</name>
+    <description>Search Mendix List views, Template Grids and Data Grids using a single-field search</description>
+    <icon/>
+    <properties>
+        <property key="entity" type="entity">
+            <caption>List view entity</caption>
+            <category>Data source</category>
+            <description>Full name of the entity "module.entity". This entity should match the list view entity</description>
+        </property>
+        <property key="values" type="object" isList="true" required="false">
+            <caption>Options</caption>
+            <category>Data source</category>
+            <description>The filter options</description>
+            <properties>
+                <property key="caption" type="translatableString" required="false">
+                    <caption>Caption</caption>
+                    <category>General</category>
+                    <description/>
+                    <translations>
+                        <translation lang="en_US">Caption</translation>
+                        <translation lang="nl_NL">Onderschrift</translation>
+                    </translations>
+                </property>
+                <property key="comparison" type="enumeration" defaultValue="static">
+                    <caption>Comparison</caption>
+                    <category>General</category>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="static">Static</enumerationValue>
```

Comment:
`Static` does not cover the real function, so filter.
Rename prop to `Filter by` enum options, `Attribute` or XPath

Reply:
Done